tests/system_info_tests.c

```diff
@@ -78,6 +80,9 @@ static int s_test_stack_trace_decoding(struct aws_allocator *allocator, void *ct
 #    if !defined(__APPLE__) /* apple doesn't always find file info */
     ASSERT_NOT_NULL(strstr(buffer, file));
 
+    /* if this is not a debug build, there may not be symbols, so the test cannot
+     * verify if a best effort was made */
+#if      defined(DEBUG_BUILD)
```

Comment:
suuuuuper trivial: indent the `# <----->  if` to match how the other proprocessor stuff is indented

Reply:
Fixed, no idea how that happened.